scripts/build.sh

```diff
@@ -19,7 +21,7 @@ GIT_COMMIT=$(./scripts/getGitCommit.sh)
 
 echo $BUILD_VERSION >$VERSION_FILE
 
-GOGC=1500 go install \
+GOGC=1500 go install -x -v \
```

Comment:
Please revert this change. `-v` was recently removed.

Reply:
I see both `-x` and `-v` as go's build options in the latest go (both in the docs as well as in go's code in master):

Docs for `-x` and `-v`: https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies

`-x` in Go's code:
https://github.com/golang/go/blob/master/src/cmd/go/internal/work/build.go#L91-L92

`-v` in Go's code:
https://github.com/golang/go/blob/master/src/cmd/go/internal/work/build.go#L86-L87
```
$ go version
go version go1.9.1 darwin/amd64

$ go help build | grep -A 1 '\-x\|\-v'
	-v
		print the names of packages as they are compiled.
	-x
		print the commands.
```

Reply:
I think you can get a better sense of time spent compiling vendor packages and netplugin code with `make compile-with-docker`, the time compiling isn't really that long, maybe there are other things happening during k8s-test?  Not sure we want to spam so much build time info for everyone, if there is a compile problem, there will be output.

Reply:
@vhosakot: The fact that Go supports both flags isn't relevant. `-x` would be far too verbose and that's not needed.

I agree only with bringing back `-v` and nothing else - no other build options.

Reply:
Can you change it to do verbose build based on the command line option & 
keep the non-verbose  build by default ?

Reply:
@rchirakk: `-v` is the only acceptable option here and it should be the default. `-x` is excessive.

Reply:
It looks like  -v was removed recently.
yes, please  add ```-v``` option & trigger the build.